eng/Versions.props

```diff
@@ -4,7 +4,6 @@
   <PropertyGroup>
     <!-- Bumping the Roslyn version used in order to ingest the new runtime source generators -->
     <UsingToolMicrosoftNetCompilers>true</UsingToolMicrosoftNetCompilers>
```

Comment:
```suggestion
```

The commit in question also added those lines.

Reply:
AFAIK (and I know very little about this :-)) - we should have one or the other - either pin fixed version, or use this directive to pull up to date compliler toolchain bits. The latter is preferreable (unless we want to manualy keep updating the pinned version).

Reply:
It adds the following: https://github.com/dotnet/arcade/blob/53cc1bc2e555aa7aea95884575d22e21d63708cf/src/Microsoft.DotNet.Arcade.Sdk/tools/Compiler.props

Not sure if we need it, if not mentioned it's just uses the one from SDK iirc

Reply:
So essentially if you don't have this line, it will just use the compiler that comes with the SDK pushed by Arcade. If you do have this line, but remove the other one this PR is removing, then it is telling arcade that you want to use the compilers coming from the compiler package that is also selected by Arcade. IIRC, the version selected by arcade, and the one coming in from the sdk provided by arcade, should be the same one, so that means that there is no real need to have this line either. My suggestion is, if removing it doesn't break the build (meaning you don't depend on a different compiler than the one in the sdk) then just remove it.